site/app/templates/forum/CreatePost.twig

```diff
@@ -0,0 +1,85 @@
+<div class="{% for class in classes %} {{ class }} {% endfor %}" id="{{ post_id }}" style="margin-left:{{ offset }}px;" reply-level="{{ reply_level }}">
+    {% if first == true %}
+        {{ title_html | raw }}
```

Comment:
Don't use `raw` unless there's a really good reason for it (this isn't one of them).

Reply:
This is linked to the `generatePostList`, I shall refactor that to avoid the requirement of raw here. Shall I do it in a separate PR or add to this one?

Reply:
Yeah, I see. This is probably something that should also be refactored so that the HTML is not generated in the View for filling this in in the generatePostList function, but may be out of scope of this issue.